renderdoc/driver/vulkan/vk_core.cpp

```diff
@@ -2792,6 +2793,10 @@ bool WrappedVulkan::ProcessChunk(ReadSerialiser &ser, VulkanChunk chunk)
       return Serialise_vkCmdSetSampleLocationsEXT(ser, VK_NULL_HANDLE, NULL);
     case VulkanChunk::vkCmdSetDiscardRectangleEXT:
       return Serialise_vkCmdSetDiscardRectangleEXT(ser, VK_NULL_HANDLE, 0, 0, NULL);
+    case VulkanChunk::DeviceMemoryRefs:
+      GetResourceManager()->Create_DeviceMemoryRefs(ser);
+      SERIALISE_CHECK_READ_ERRORS();
```

Comment:
Related to the other comment about refactoring how this chunk is serialised, with those changes this should follow suit and the `SERIALISE_CHECK_READ_ERRORS()` should be in the usual place in `Serialise_DeviceMemoryRefs()` after the value serialisation.